addons/website_slides/static/src/js/slides_course_fullscreen_player.js

```diff
@@ -493,7 +493,7 @@
      *
      * This widget is rendered sever side, and attached to the existing DOM.
      */
-    var Fullscreen = SlideCoursePage.extend({
+    export var Fullscreen = SlideCoursePage.extend({
```

Comment:
This is already default-exported, let's avoid making more diff in other modules than necessary